app/javascript/packages/document-capture/components/acuant-selfie-capture-canvas.jsx

```diff
@@ -8,6 +8,7 @@ function FullScreenLoadingSpinner({ fullScreenRef, onRequestClose, fullScreenLab
     <FullScreen ref={fullScreenRef} label={fullScreenLabel} onRequestClose={onRequestClose}>
       <img
         src={getAssetPath('loading-badge.gif')}
+        aria-label="loading spinner"
```

Comment:
I needed to add this so I could find it in tests

Reply:
This is content that could be exposed to end-users using assistive technology, and may be redundant with the `alt` attribute. I'd suggest assigning this as the `alt` attribute if we're okay with the content being used as a text alternative for the image, and ensure we have translations for it.

Reply:
Modifying the component code to get the tests to work wasn't the right thing here. I revised [here](https://github.com/18F/identity-idp/pull/9676/commits/2293cf0c12a18dbf1673cadce9c29a21040794f2).